robyn/processpool.py

```diff
@@ -182,7 +182,7 @@ def spawn_process(
     server.set_response_headers_exclude_paths(excluded_response_headers_paths)
 
     for route in routes:
-        route_type, endpoint, function, is_const = route
+        route_type, endpoint, function, is_const, auth_required, openapi_name, openapi_tags = route
```

Comment:
One question @dave42w 👋 

Where are the `auth_required`, `openapi_name` , etc used here? Could we do a better job in unpacking if now?

Reply:
It's preparation for the full route refactor. Then the authentication route processing happens in server start rather than in add_route.
I wanted to reduce the times I need to touch the code and so put this in before it's needed.